tests/installation/bootloader_svirt.pm

```diff
@@ -134,8 +134,19 @@ sub run {
     my $hdddir = "$basedir/openqa/${share_factory}hdd $basedir/openqa/${share_factory}hdd/fixed";
     my $size_i = get_var('HDDSIZEGB', '10');
     foreach my $n (1 .. get_var('NUMDISKS')) {
-        if (my $hdd = get_var('HDD_' . $n)) {
+        if (my $hdd = fileparse(get_var('HDD_' . $n))) {
```

Comment:
don't we need to use `is_vmware` here ? I would assume this code was used by some other not vmware related tests , right ?

Reply:
Yes, this part should be general for both (xen and vmware) hosts, in case one would like to add more drives or boot from one as we do.

Reply:
It might be for both, but then you are using `vmware_openqa_datastore` below, which is only related to vmware only, right? Or maybe I'm missing something.

Reply:
Exactly, only the location of the assets changes here.

Reply:
yep but if `check_var('VIRSH_VMM_FAMILY', 'vmware')` is `false`, `$vmware_openqa_datastore` will be `undef` and below statements will fail, won't they?

Reply:
I can not understand the use case . I just checked  https://perldoc.perl.org/File/Basename.html#:~:text=To%20be%20safe%2C%20if%20you,fileparse()%20's%20behaviour. and if I get it right `fileparse` main use case is `/some/long/path/to_file.txt` than it will split it in several parts so normally you will do something like this : 
` my($filename, $dirs, $suffix) = fileparse($path);`
On the other hand usually `HDD_*` contains just filename and if you will give fileparse just a filename you will get back just a filename.  I checked all VRs which you did and in all cases HDD_1 does not change anyhow in this line 
Can you please trigger some job which would actually use this line ?

Reply:
> I can not understand the use case . I just checked https://perldoc.perl.org/File/Basename.html#:~:text=To%20be%20safe%2C%20if%20you,fileparse()%20's%20behaviour. and if I get it right `fileparse` main use case is `/some/long/path/to_file.txt` than it will split it in several parts so normally you will do something like this :
> ` my($filename, $dirs, $suffix) = fileparse($path);`
> On the other hand usually `HDD_*` contains just filename and if you will give fileparse just a filename you will get back just a filename. I checked all VRs which you did and in all cases HDD_1 does not change anyhow in this line
> Can you please trigger some job which would actually use this line ?

```
HDD_1 : "/var/lib/openqa/pool/10/SLES15-SP3-JeOS.x86_64-15.3-MS-HyperV-Build20.16.vhdx.xz"
```
https://openqa.suse.de/tests/4693043/file/vars.json

Reply:
And all 3 VRs are using this line.

Reply:
> yep but if `check_var('VIRSH_VMM_FAMILY', 'vmware')` is `false`, `$vmware_openqa_datastore` will be `undef` and below statements will fail, won't they?

Could you point me to which statements are you referring to, please ? Seems like we have more problems in this thread :)

Reply:
ok I was not aware that HDD_1 is updated with absolute path . now it makes perfect sense

Reply:
here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/aa0ad2051e7dc8ce0fb493cc1054f205549076c7/tests/installation/bootloader_svirt.pm#L141

Reply:
> here:
> 
> https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/aa0ad2051e7dc8ce0fb493cc1054f205549076c7/tests/installation/bootloader_svirt.pm#L141

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/aa0ad2051e7dc8ce0fb493cc1054f205549076c7/tests/installation/bootloader_svirt.pm#L139

Should be covered by compressed vmdk image (VMware format)